optimum/habana/transformers/models/mllama/modeling_mllama.py

```diff
@@ -908,6 +908,12 @@ def forward(
         )
 
         hidden_states = outputs[0]
+        _, seq_len, _ = hidden_states.shape
+        if seq_len > 1 and not self.training:
```

Comment:
@ANSHUMAN87 Does your change also require adding `trim_logits` to the forward interface? and do below modification? 
`if seq_len > 1 and trim_logits and not self.training:`

Reply:
I agree with @yafshar, I'm curious about this too :eyes: